src/dotnet/commands/dotnet-msbuild/MSBuildForwardingApp.cs

```diff
@@ -31,7 +31,7 @@ public class MSBuildForwardingApp
             };
         
         private readonly IEnumerable<string> _msbuildRequiredParameters = 
-            new List<string> { "/m", "/v:m" };
+            new List<string> { "/m" };
```

Comment:
Issue #5110 only talks about `dotnet build`. I do not think we should change the verbosity for `run`, `restore`, etc. @blackdwarf ?